### PR TITLE
Stop recording annotations set to false

### PR DESCRIPTION
**Pull request by rhatdan**
False is the assumed value, and inspect and podman generate kube are being cluttered with a ton of annotations that indicate nothing.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
podman container inspect and podman kube generate will no longer list default annotations set to false.
```

### CODE REVIEW

**Comment by rhatdan** [comment]
Gives cleaner looking kube.yaml.
```
./bin/podman create --name dan --replace fedora echo hi
./bin/podman generate kube dan
# Save the output of this file and use kubectl create -f to import
# it into Kubernetes.
#
# Created with podman-4.4.0-dev
apiVersion: v1
kind: Pod
metadata:
  creationTimestamp: "2022-12-19T21:10:01Z"
  labels:
    app: dan-pod
  name: dan-pod
spec:
  containers:
  - command:
    - echo
    - hi
    env:
    - name: QEMU_CPU
      value: max
    image: registry.fedoraproject.org/fedora:latest
    name: dan
```
Versus
```
/bin/podman create --name dan --replace fedora echo hi
$ /bin/podman generate kube dan
# Save the output of this file and use kubectl create -f to import
# it into Kubernetes.
#
# Created with podman-4.3.1
apiVersion: v1
kind: Pod
metadata:
  annotations:
    io.kubernetes.cri-o.TTY/dan: "false"
    io.podman.annotations.autoremove/dan: "FALSE"
    io.podman.annotations.init/dan: "FALSE"
    io.podman.annotations.privileged/dan: "FALSE"
    io.podman.annotations.publish-all/dan: "FALSE"
  creationTimestamp: "2022-12-19T21:11:51Z"
  labels:
    app: dan-pod
  name: dan-pod
spec:
  automountServiceAccountToken: false
  containers:
  - command:
    - echo
    - hi
    env:
    - name: QEMU_CPU
      value: max
    image: registry.fedoraproject.org/fedora:latest
    name: dan
    securityContext:
      capabilities:
        drop:
        - CAP_MKNOD
        - CAP_NET_RAW
        - CAP_AUDIT_WRITE
  enableServiceLinks: false
```

**Review by vrothberg — COMMENTED** [review]
LGTM
@containers/podman-maintainers PTAL

**Review by giuseppe — APPROVED** [review]
/lgtm
/hold

**Comment by umohnani8** [comment]
LGTM
/hold cancel

**Pull request merged**
